go.mod

```diff
@@ -1,13 +1,12 @@
 module github.com/container-orchestrated-devices/container-device-interface
 
-go 1.17
+go 1.18
```

Comment:
Note that podman uses 1.17. Is this a hard requirement?

Reply:
No, not a hard requirement, I guess. But IMO there is no reason to stay on 1.17, especially since it has been [EOL'd](https://endoflife.date/go) more than 3 months ago. 1.18 has been released more than 6 months ago and also 1.19 more than 3 months ago.

Reply:
OK, that's fair. I have created https://github.com/containers/podman/pull/16530 to update the version in podman.

Reply:
Yes, definitely we can. I can switch it back to 1.17 and we can bump it later, preferably directly to 1.19 if all of our downstream/potential downstream has switched.

Reply:
Pushed again with the go version bump patch dropped.

Reply:
Thanks. I'll follow up on the podman PR next week and we can revisit once that's sorted out.